src/oasis-nexus/api.ts

```diff
@@ -876,6 +894,11 @@ export const useGetRuntimeEvents: typeof generated.useGetRuntimeEvents = (
                         event.body.amount.Denomination ||
                         getTokensForScope({ network, layer: runtime })[0].ticker,
                     },
+                    owner:
+                      event.body?.owner && event.body.owner === oasisAddress ? params?.rel : event.body.owner,
+                    from:
+                      event.body?.from && event.body.from === oasisAddress ? params?.rel : event.body.from,
+                    to: event.body?.to && event.body.to === oasisAddress ? params?.rel : event.body.to,
```

Comment:
These should be saved into owner_eth, from_eth, to_eth fields if they don't exist

Reply:
These don't run for consensus_accounts.undelegate_start event, because this code block was meant to normalize `Amount` field. undelegate_start only has `Shares` field.

I'd rather move it into a block that needlessly runs for all events

Reply:
thx